addons/docs/src/frameworks/svelte/prepareForInline.ts

```diff
@@ -2,8 +2,8 @@ import { AnyFramework, StoryFn } from '@storybook/csf';
 
 import React from 'react';
 
-// @ts-ignore
-import HOC from './HOC.svelte';
```

Comment:
See point above

<details>

> So the svelte app, has a `XYZ.svelte` decorator, which we do not parse/compile/transpile at all. To us, it's NOT source.
> 
> before this file was copied over (multiple times) to all the `dist` folders (duplicated).
besides being inefficient, and increasing the size of our packages (very minuscule), we can just point to a file relative to the package root, because why have babel copy the file 3 times if it can just exist in 1 place instead?
</details>